app/cypress/e2e/0-ndr-core-tests/auth_routes/auth_general_browser_states.cy.js

```diff
@@ -1,5 +1,8 @@
 import authPayload from '../../../fixtures/requests/auth/GET_TokenRequest_GP_ADMIN.json';
 import { Roles } from '../../../support/roles';
+import dbItem from '../../../fixtures/dynamo-db-items/active-patient.json';
+import { pdsPatients, stubPatients } from '../../../support/patients';
```

Comment:
My IDE is suggesting this is an unused import. Can we remove it if that is the case?